www/js/services.js

```diff
@@ -0,0 +1,38 @@
+angular.module('open311.services', [])
+.factory('API', function($http, $httpParamSerializer, $q) {
+
+    var recentCasesData = [];
+
+    var getRequests = function(lat, long) {
+      var deferred = $q.defer();
+
+      var params = $httpParamSerializer({
+          lat: lat,
+          long: long 
+      });
+
+      $http.get('https://seeclickfix.com/open311/v2/requests.json?' + params)
+        .then(function(response) {
+          recentCasesData = response.data;
+          deferred.resolve(response);
```

Comment:
You don't need to use another `deferred` here, just `return` the response, since you're already inside a Promise.

```js
.then(function(response) {
    recentCasesData = response.data
    return response;
})
```